File: Assignement 13/SumEven.c
#include<stdio.h>

void SumEven(int iNo)
{
    int iSum=0;
    for(int iCnt=1;iCnt<=iNo;iCnt++)
    {
        if(iCnt%2==0)
        {
            iSum=iSum+iCnt;
        }
    }
    printf("%d",iSum);

}
int main()
{
    int iValue=0;

    printf("Enter Range:");
    scanf("%d",&iValue);

    SumEven(iValue);

    return 0;
}

File: Assignment 20/SumEven.java
class logic
{
    void Even(int No)
    {
        int iSum=0;
        for(int i=1;i<=No;i++)
        {
            if(i%2==0)
            {
                iSum=iSum+i;
            }
        }
        System.out.println(iSum);
    }

}

class SumEven
{
    public static void main(String a[])
    {
        logic lobj=new logic();
        lobj.Even(10);
    }
}